newrelic-php-agent/README.md

```diff
@@ -20,6 +20,36 @@ The following table lists the configurable parameters of the newrelic-php-agent
 |  Parameter | Description | Default |
 | --- | --- | --- |
 | `image.repository` | The image repository to pull from | `chatwork/newrelic-php-agent`|
-| `image.tag` | The image tag to pull | `latest`|
+| `image.tag` | The image tag to pull | `0.0.1`|
 | `image.imagePullPolicy` | Image pull policy | `IfNotPresent` |
-| `image.command` | Image Entrypoint | `[]` |
\ No newline at end of file
+| `image.command` | Image Entrypoint | `[]` |
+| `image.pullSecrets` | Image PullSecrets | `[]` |
+|  `address` | Use `sockfile` if this value is a file path. Otherwise, use `containerPort`. input in the format `host:port`.  | `/tmp/.newrelic.sock` |
+|  `port` | When `sockfile` is not used, set `containerPort`  | `nil` |
+|  `livenessProbe` | Overrides the default liveness probe | tcpSocket port `31339` |
+|  `readinessProbe` | Overrides the default readness probe | tcpSocket port `31339` |
+|  `daemonset.enabled` | Enables daemonset | `TRUE` |
+|  `daemonset.useHostNetwork` | If true, use the host's network | `FALSE` |
+|  `daemonset.extraPorts` | Extra ports | `[]` |
+|  `daemonset.extraEnv` | Extra env | `[]` |
+|  `daemonset.extraEnvFrom` | Extra env form | `[]` |
+|  `daemonset.extraVolumes` | Extra volumes | `[]` |
+|  `daemonset.extraVolumeMounts` | Extra volume mounts | `[]` |
+|  `daemonset.tolerations` | Set Pod tolerations | `[]`|
+|  `daemonset.affinity` | Set Pod affinity | `[]`|
+|  `daemonset.nodeSelector` | Set Pod nodeSelector | `[]`|
+|  `deployment.enabled` | Enables deployment | `FALSE` |
+|  `deployment.useHostNetwork` | If true, use the host's network | `FALSE` |
+|  `deployment.replicaCount` | Deployment replicaCount | `1` |
+|  `deployment.extraPorts` | Extra ports | `[]` |
+|  `deployment.extraEnv` | Extra env | `[]` |
+|  `deployment.extraEnvFrom` | Extra env form | `[]` |
+|  `deployment.extraVolumes` | Extra volumes | `[]` |
+|  `deployment.extraVolumeMounts` | Extra volume mounts | `[]` |
+|  `deployment.tolerations` | Set Pod tolerations | `[]`|
+|  `deployment.affinity` | Set Pod affinity | `[]`|
+|  `deployment.nodeSelector` | Set Pod nodeSelector | `[]`|
+|  `ingress.enabled` | Enables Ingress | `FALSE` |
```

Comment:
@cw-niinuma 
https://github.com/chatwork/charts/pull/118/files#diff-93431a1dbde9948c8743e81cd795cb05R12
Ingress settings are missing from the README.